### PR TITLE
Additional parsing functions and tests for Contentful content followi…

### PR DESCRIPTION
**Pull request by mcstover**
…ng our page structure.

### CODE REVIEW

File: test/unit/specs/util/contentfulUtils.spec.js [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+				}
+			}
+		};
+
+		test('should return flattened page, layout, and content group data', () => {
+			expect(processPageContentFlat(ccPageRaw)).toMatchObject(pageLevelLayoutAndContentGroups);
+		});
+	});
+
+	describe('processPageContentFlattened with non specific data', () => {
```
Comment (mcstover) [inline test/unit/specs/util/contentfulUtils.spec.js:197]: @emuvente @eddieferrer Kinda feel like I should reduce all the other tests to follow this paradigm...

**Comment by mcstover** [comment]
Just building some of this out as I go on the landing page work and intend to improve and refine it over time. I have a couple other PR's coming which depend on this so I want to get these foundations in place.

**Comment by eddieferrer** [comment]
This is great, much cleaner than before. 👍

**Review by emuvente — APPROVED** [review]
👍

**Pull request merged**